tests/test_check_metadata_job.py

```diff
@@ -0,0 +1,176 @@
+"""Module for testing the CheckMetadataJob class."""
+
+import unittest
+import tempfile
```

Comment:
You might run into issues running unit tests on other systems if you use tempfile and shutil copy. Since nothing is being written out, it's unclear why we need these packages.